After 2.10 Collections can use vars plugins (#77216)
ansible/ansible: docs/docsite/rst/dev_guide/developing_collections_structure.rst

Add a 'per plugin type' specific subdirectory here, including ``module_utils`` which is usable not only by modules, but by most plugins by using their FQCN. This is a way to distribute modules, lookups, filters, and so on without having to import a role in every play.

> Vars plugins are supported in collections as long as they require being explicitly enabled (using ``REQUIRES_ENABLED``) and they are included using their fully qualified collection name. See :ref:`enable_vars and :ref:`developing_vars_plugins` for details. Cache plugins may be used in collections for fact caching, but are not supported for inventory plugins.

Reviewer: ```suggestion
Vars plugins are supported in collections as long as they require being explicitly enabled (using ``REQUIRES_ENABLED``) and they are included using their fully qualified collection name. See :ref:`enable_vars` and :ref:`developing_vars_plugins` for details. Cache plugins may be used in collections for fact caching, but are not supported for inventory plugins.
```

Add a 'per plugin type' specific subdirectory here, including ``module_utils`` which is usable not only by modules, but by most plugins by using their FQCN. This is a way to distribute modules, lookups, filters, and so on without having to import a role in every play.

> Vars plugins were initially unsupported in collections, but starting in Ansible 2.10 they are supported as long as they require being explicitly enabled (using ``REQUIRES_ENABLED``) and they are included using their fully qualified collection name. See `vars plugins documentation`_ and `developing vars plugins`_ for details. Cache plugins may be used in collections for fact caching, but are not supported for inventory plugins.

Reviewer: ```suggestion
Vars plugins are supported in collections as long as they require being explicitly enabled (using ``REQUIRES_ENABLED``) and they are included using their fully qualified collection name. See :ref:`enable_vars and :ref:`developing_vars_plugins` for details. Cache plugins may be used in collections for fact caching, but are not supported for inventory plugins.
```

> .. _vars plugins documentation: https://docs.ansible.com/ansible/latest/plugins/vars.html#enabling-vars-plugins

Reviewer: ```suggestion
```

> .. _developing vars plugins: https://docs.ansible.com/ansible/latest/dev_guide/developing_plugins.html#vars-plugins

Reviewer: ```suggestion
```
